package.json

```diff
@@ -30,26 +32,45 @@
   "scripts": {
     "start": "stripes serve",
     "lint": "eslint .",
-    "test": "echo 'placeholder. no tests implemented'"
+    "test": "stripes test karma"
   },
   "devDependencies": {
-    "@folio/eslint-config-stripes": "^5.1.0",
-    "@folio/stripes": "^4.0.0",
     "@folio/stripes-cli": "^1.3.1",
```

Comment:
I saw @mkuklis [add support for `type: "handler"`](https://github.com/folio-org/stripes-cli/pull/231) in stripes-cli so we should use that value above and depend on a compatible version of `"@folio/stripes-cli": "^1.18.0"` here.